modules/nf-scil/image/powderaverage/meta.yml

```diff
@@ -0,0 +1,53 @@
+---
+name: "image_powderaverage"
+description: Calculate the powder average of a 4D diffusion volume.
+keywords:
+  - DWI
+  - Powder Average
+  - BET
+tools:
+  - "scilpy":
+      description: "The Sherbrooke Connectivity Imaging Lab (SCIL) Python dMRI processing toolbox."
+      homepage: "https://github.com/scilus/scilpy.git"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+
+  - dwi:
+      type: file
+      description: Nifti DWI volume to compute the powder average on.
+      pattern: "*.nii.gz"
+
+  - bval:
+      type: file
+      description: b-values file.
+      pattern: "*.bval"
+
+  - mask:
+      type: file
+      description: Binary mask to apply on the DWI volume.
+      pattern: "*.nii.gz"
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+  - versions:
```

Comment:
just reverse version and pwd_avg so that they follow the same order as outputs and add spaces between files as for inputs